src/components/DataCollector/activity/click.js

```diff
@@ -96,9 +96,9 @@ const createClickHandler = (config, logger, collect) => {
 };
 
 export default (config, logger, collect) => {
-  const enabled = config.get("dataCollector.clickActivity.enabled", true);
+  const enabled = config.get("clickCollectionEnabled");
```

Comment:
Let' get rid of `getNestedObject` & `setNestedObject`.

Reply:
:+1: